src/components/Comments/WriteComment.js

```diff
@@ -74,7 +74,7 @@ const WriteComment = ({
     <View style={isViewOnly ? styles.isViewOnlyColumnContainer : styles.columnContainer}>
       <View style={styles.connectorTop(inReplyCommentView, showProfileConnector)} />
       <View style={{flexDirection: 'row', paddingRight: 20, paddingLeft: 7}}>
-        <Text style={styles.replyToContainer(inReplyCommentView)}>
+        <Text style={styles.replyToContainer}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a dynamic `marginLeft` in `replyToContainer` to a static value may affect the layout depending on the `inReplyCommentView` state. If this state is important for the layout, consider reverting this change.
```diff
-    marginLeft: 60,
+    marginLeft: inReplyCommentView ? 90 : 60,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->